packages/react-dom/src/events/BeforeInputEventPlugin.js

```diff
@@ -462,6 +462,8 @@ function extractBeforeInputEvent(
 const BeforeInputEventPlugin = {
   eventTypes: eventTypes,
 
+  isLegacy: true,
```

Comment:
I'm curious how this flag relates to interactive updates and if this will end up being a more general "noPassive" flag or something.